pkg/scyllaclient/client_scylla.go

```diff
@@ -494,36 +506,25 @@ func repairStatusShouldRetryHandler(err error) *bool {
 	return nil
 }
 
-// RepairStatus returns current status of a repair command.
-// If waitSeconds is bigger than 0 long polling will be used.
-// waitSeconds argument represents number of seconds.
-func (c *Client) RepairStatus(ctx context.Context, host, keyspace string, id int32, waitSeconds int) (CommandStatus, error) {
-	ctx = customTimeout(forceHost(ctx, host), c.longPollingTimeout(waitSeconds))
+// RepairStatus waits for repair job to finish and returns its status.
+func (c *Client) RepairStatus(ctx context.Context, host string, id int32) (CommandStatus, error) {
+	ctx = forceHost(ctx, host)
 	ctx = withShouldRetryHandler(ctx, repairStatusShouldRetryHandler)
-
+	ctx = noTimeout(ctx)
```

Comment:
It should be configurable via repair configuration kept in scylla-manager.yaml.

@asias what is the expected time for the repair job to complete ? 
By repair job I mean job created by the call to POST /storage_service/repair_async/{keyspace}.

I'm looking for some safe timeout that manager would introduce on calling for the job status.

Reply:
Removing noTimeout() from the current implementation.
Please don't use it.
This context is passed to many different calls as well.